spyder_kernels/utils/nsview.py

```diff
@@ -632,8 +650,18 @@ def get_supported_types():
     in spyder-docs
     """
     from datetime import date, timedelta
-    editable_types = [int, float, complex, list, set, dict, tuple, date,
-                      timedelta, str]
+    editable_types = [
+        int,
+        float,
+        complex,
+        list,
+        set,
+        frozenset,
+        dict,
+        tuple,
+        date,
+        timedelta,
+        str]
```

Comment:
```suggestion
        str,
    ]
```

Small formatting improvement.